src/common/scaffold/scaffold.jsx

```diff
@@ -189,32 +166,32 @@ export default function Scaffold() {
         debounce: 500,
     });
 
-
     const school = useSelector((state) => state.payment.payment.school);
```

Comment:
Nit picking;

Line 148                 if (result.status === 200 && result.data === 'Greatings for FINTLabs :)') { 

should be Greetings

Reply:
Made changes to front and back to fix spelling error.